ush/python/pygfs/utils/marine_da_utils.py

```diff
@@ -0,0 +1,105 @@
+import f90nml
+import os
+from logging import getLogger
+import xarray as xr
+
+from wxflow import FileHandler, logit, WorkflowException, AttrDict, parse_j2yaml, jinja
+
+logger = getLogger(__name__.split('.')[-1])
+
+
+@logit(logger)
+def run(exec_cmd):
```

Comment:
We will try to elevate this to `wxflow.Executable` for a short-hand to use by all.  
This is repeated in many places.

Reply:
ok.